### ISSUE TITLE
Does quiescent immediately render into the DOM?

### ISSUE THREAD

**Issue by samedhi** [levand/quiescent#28]
(From the documentation example)
....
(q/render (ArticleList my-data)
          (.getElementById js/document "root-element"))
....

Is it the case that as soon as q/render has returned, that the changes have been fully applied to the DOM (synchronously)? I believe at this point that both Reagent and Om render using the RequestAnimationFrame, and as such are effectively asynchronous with respect to the actual React in memory DOM being applied to the actual DOM.

If this is the case, I would mention this somewhere, as it is a feature that some people might be looking for in a renderer.

For instance, I have the case where I often render something into the DOM, take a measurement from it, and then adjust certain things based on the measurement. For me, a synchronous renderer is easier to reason about than an asynchronous one.

tldr; Is quiscent a synchronous renderer; can I be certain that everything has "been done" after q/render returns?
